flocker/provision/_aws.py

```diff
@@ -102,45 +271,18 @@ def aws_provisioner(access_key, secret_access_token, keyname,
     :param list security_groups: List of security groups to put created nodes
         in.
     """
-    # Import these here, so that this can be imported without
-    # installing libcloud.
-    from libcloud.compute.providers import get_driver, Provider
-    driver = get_driver(Provider.EC2)(
-        key=access_key,
-        secret=secret_access_token,
-        region=region)
-
-    location = [loc for loc in driver.list_locations()
-                if loc.availability_zone.name == zone][0]
-
-    def create_arguments(disk_size):
-        return {
-            "location": location,
-            "ex_securitygroup": security_groups,
-            "ex_blockdevicemappings": [
-                {"DeviceName": "/dev/sda1",
-                 "Ebs": {"VolumeSize": disk_size,
-                         "DeleteOnTermination": True,
-                         "VolumeType": "gp2"}}
-            ],
-            # On some operating systems, a tty is requried for sudo.
-            # Since AWS systems have a non-root user as the login,
-            # disable this, so we can use sudo with conch.
-            "ex_userdata": dedent("""\
-                #!/bin/sh
-                sed -i '/Defaults *requiretty/d' /etc/sudoers
-                """)
-        }
-
-    provisioner = LibcloudProvisioner(
-        driver=driver,
+    # Import these here, so that this can be imported without installng
```

Comment:
No longer relevant comment.

Can we just expect that all dev requirements will be available?

Existing typo "installng" > "installing".